tutorials/developing_with_ign-cmake.md

```diff
@@ -96,26 +96,10 @@ To change the build system type, set the CMake flag:
 -GNinja
 ```
 
-### Address sanitizer (ASan)
+### Build sanitizers
 
-The `gcc` and `clang` compilers have a set of flags to generate instrumented builds for detecting memory leaks.
-
-By default, address sanitizer is *not used*.
-
-To enable address sanitizer, set all of the following flags:
-
-```
--DCMAKE_CXX_FLAGS="-fsanitize=address -fsanitize=leak -g"
--DCMAKE_C_FLAGS="-fsanitize=address -fsanitize=leak -g"
--DCMAKE_EXE_LINKER_FLAGS="-fsanitize=address -fsanitize=leak"
--DCMAKE_MODULE_LINKER_FLAGS="-fsanitize=address -fsanitize=leak"
-```
-
-This will report if memory is leaked during execution of binaries or tests.
-
-More information about address santizier can be found in the [ASan documentation](https://github.com/google/sanitizers/wiki/AddressSanitizer).
-
-Note: Address sanitizer may have an impact on the performance of execution.
+`IGN_SANITIZER` CMake parameter can be used with different compilers to support the detection of different problems in the code.
+[Check the documentation for `IGN_SANITIZER` flag](ign_cmake_sanitizers.md)
```

Comment:
The problem we have is that the `ign_cmake_sanitizers` is not really a tutorial but more a documentation on how to use the feature. Not sure if we can consider it as a "tutorial". It is referenced from the "Developing with Ignition CMake"  and place inside the tutorials since I did not find a better place for it.

Reply:
thanks @azeey and @chapulina for the information. I added the entry in [c55d115](https://github.com/ignitionrobotics/ign-cmake/pull/210/commits/c55d115a09c52389a21cf56fb352d1d8cd910399)